UI/Source/AppKitExtensions/NSCollectionView+Selection.swift

```diff
@@ -3,30 +3,30 @@ import Foundation
 
 public extension NSCollectionView {
 
-    /// Sets the selection to the first item in the collection view.
-    public func selectFirstItem() {
+    /// Sets the selection to the first item in the collection view, and calls the delegate's selection method.
```

Comment:
should probably adjust the comment as well. maybe adding "if notifyingDelegate is true"